client/state/sites/reducer.js

```diff
@@ -9,7 +9,8 @@ import keyBy from 'lodash/keyBy';
  * Internal dependencies
  */
 import { plans } from './plans/reducer';
-import { domains } from './domains/reducer';
+import domains from './domains/reducer';
```

Comment:
yeah, I did a PR yesterday to fix that changing the way that the site-domains reducer is exported replacing `default` by `domains`:

```es6
const const domains combineReducers( { // ... } );
```

but I'm not sure what should be out convention in this sense. For instance above we can import plans reducers with `{ plans }` but we do `mediaStorage` directly.